telephony/java/android/telephony/TelephonyManager.java

```diff
@@ -1935,6 +1935,13 @@ public CellLocation getCellLocation() {
         }
     }
 
+    /**
+     * @hide
+     */
+    public CellLocation getCellLocationBySubId(int subId) {
```

Comment:
Are you planning to call those one day?

Reply:
Ah you said the blobs may try. Ignore me

Reply:
Exactly, I had someone send me a log once and this fixed their issue because blobs were old.